martin/build.rs

```diff
@@ -97,8 +19,16 @@ fn webui() {
 
     // TODO: we may need to move index.html one level down per change_detection() docs
     static_files::NpmBuild::new(martin_ui_dir)
-        .target(&target_to_keep)
-        .change_detection();
+        .node_modules_strategy(Default::default())
```

Comment:
once this is pub api

```suggestion
        .node_modules_strategy(static_files::NodeModulesStrategy::MoveToOutDir)
```